src/logic/basic.lean

```diff
@@ -1136,6 +1136,9 @@ by simp only [← @forall_eq _ p a, ← forall_and_distrib, ← or_imp_distrib,
 @[simp] theorem forall_eq_or_imp {a' : α} : (∀ a, a = a' ∨ q a → p a) ↔ p a' ∧ ∀ a, q a → p a :=
 by simp only [or_imp_distrib, forall_and_distrib, forall_eq]
 
+lemma ne.ne_or_ne {x y z : α} (h : x ≠ y) : x ≠ z ∨ y ≠ z :=
```

Comment:
I think I would prefer `z` to be explicit. So that I can do
```
obtain (hz|hz) := h.ne_or_ne z,
```

Reply:
I agree: `z` is explicit now!